packages/upload-api/src/customer/get.js

```diff
@@ -24,11 +24,11 @@ const get = async ({ capability }, context) => {
     return { error: new UnknownProvider(capability.with) }
   }
 
-  const customer = await context.provisionsStorage.getCustomer(
+  const result = await context.provisionsStorage.getCustomer(
     capability.with,
     capability.nb.customer
   )
-  return { ok: { customer: customer.ok ? { did: customer.ok.did } : null } }
+  return result.ok ? { ok: { did: result.ok.did } } : result
```

Comment:
Just flagging that this is a breaking change, and we need to make sure nothing expects this to never fail.

Reply:
ah good call, I'll make this a breaking `feat`. I can't find anywhere that we actually use this capability at the moment, do you remember if we use this anywhere?